media-libs/raptor/raptor-2.0.15.ebuild

```diff
@@ -64,4 +64,8 @@ src_install() {
 	# https://bugs.gentoo.org/467768
 	local _rdocdir=/usr/share/doc/${PF}/html/${MY_PN}
 	[[ -d ${ED}/${_rdocdir} ]] && dosym ${_rdocdir} /usr/share/gtk-doc/html/${MY_PN}
+
+	#Needed by packages expecting this:
+	dosym /usr/include/raptor2/raptor.h /usr/include/raptor.h
+	dosym /usr/include/raptor2/raptor2.h /usr/include/raptor2.h
```

Comment:
Not sure what to do here. The error hydrogen outputs is: 

/usr/include/lrdf.h: fatal error: raptor.h: No such file or directory. 

Am looking into it, but advice would be appreciated. 

Emerged another pkg that depends on liblrdf without issues, so it does seem like hydrogen is the problem.